prober/http.go

```diff
@@ -309,7 +309,8 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 
 	httpConfig := module.HTTP
 
-	if !strings.HasPrefix(target, "http://") && !strings.HasPrefix(target, "https://") {
+	targetLower := strings.ToLower(target)
+	if !strings.HasPrefix(targetLower, "http://") && !strings.HasPrefix(targetLower, "https://") {
```

Comment:
thanks for the PR, can you please add a test so don't break this accidently in the future :)